src/WindowManager.vala

```diff
@@ -873,6 +873,8 @@ namespace Gala
 
 				ui_group.add_child (old_actor);
 
+				maximizing.add (actor);
```

Comment:
Previously `maximizing` list object wasn't used at all, we now actually add the actor to the maximizing list.